docs/components/iam/getting-started/docker/setup-environment.md

```diff
@@ -0,0 +1,141 @@
+
+```yaml
+version: "3.6"
+
+services:
+  iam:
+    image: camunda/iam:latest
+    ports:
+      - 8080:8080
+    environment:
+      IAM_CLIENT_SECRET: [a random 32 char alphanumeric string]
+
+      CLIENTS_0_BASE_URL: http://localhost:18085
+      CLIENTS_0_CLIENT_ID: test-client
+      CLIENTS_0_NAME: Test Client
+      CLIENTS_0_LOGOUT_URL: http://localhost:18085/logout_callback
+      CLIENTS_0_CLIENT_SECRET: [a random 32 char alphanumeric string]
+      CLIENTS_0_DELETE_VALIDATE_ENDPOINT: http://localhost:18085/users/user-can-be-deleted/
+      CLIENTS_0_DELETE_ENDPOINT: http://localhost:18085/users/
+
+      ENFORCE_HTTPS: "false"
+```
+
+:::note
+Here we set `ENFORCE_HTTPS` to false, so we can use localhost. We would recommend removing this option prior to production use.
+:::
+
+IAM requires a database to function, let's add a database service to our docker-compose.yml file:
+
+```yaml
+database:
+    image: postgres:13.3-alpine
+    environment:
+    POSTGRES_DB: iam
+    POSTGRES_USER: camunda
+    POSTGRES_PASSWORD: [a random alphanumeric string]
+    ports:
+      - 15432:5432
+    healthcheck:
+        test: pg_isready -d iam -U camunda
+        interval: 30s
+        timeout: 15s
+        retries: 5
+```
+
+:::caution
+The IAM application currently only supports PostgreSQL 12+
+:::
+:::caution
+The IAM application generates an encryption key per start, this means the database needs to be recreated each time.
+:::
+
+We will also need to add new entries to the `services.iam.environment` section to tell IAM where the database is located and the password for access:
+
+```yaml
+  DB_PASSWORD: [the password you entered for `database.POSTGRES_PASSWORD`]
+  DB_URL: jdbc:postgresql://database:5432/iam
+```
+
+Next, we will tell Docker Compose that the `iam` service is dependent on the `database` service by adding the following lines under `services.iam`
+
+```yaml
+    depends_on:
+      - database
+```
+
+And finally, we'll add an override to enable the user management functionality. To do this, add the following line to the `services.iam.environment` section:
+
+```yaml
+  FEATURE_USER_MANAGEMENT: "true"
+```
+
+Your `docker-compose.yml` file should now look like this
+
+<details><summary>Show complete Docker Compose file</summary>
+
+```yaml
+version: "3.6"
+
+services:
+  application:
+    image: camunda/iam:latest
+    depends_on:
+      - database
+    ports:
+      - 8080:8080
+    environment:
+      IAM_CLIENT_SECRET: [a random 32 char alphanumeric string]
+
+      CLIENTS_0_BASE_URL: http://localhost:18085
+      CLIENTS_0_CLIENT_ID: test-client
+      CLIENTS_0_NAME: Test Client
+      CLIENTS_0_LOGOUT_URL: http://localhost:18085/logout_callback
+      CLIENTS_0_CLIENT_SECRET: [a random 32 char alphanumeric string]
+      CLIENTS_0_DELETE_VALIDATE_ENDPOINT: http://localhost:18085/users/user-can-be-deleted/
+      CLIENTS_0_DELETE_ENDPOINT: http://localhost:18085/users/
```

Comment:
Do we need the default client configuration although nothing is started at port `18085`? If I understood it correctly, the application specific configuration for IAM will be added to that application docs, so I would omit it here.